src/Main.js

```diff
@@ -1,33 +1,81 @@
 import { Component } from 'react';
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Image from 'react-bootstrap/Image'
+import Card from 'react-bootstrap/Card'
 
 class Main extends Component {
   render(){
+
+    const objs = this.props.beastObj;
+
     return (
-      <div>
-        <h1>Beasts: {this.props.message}</h1>
-        <HornedBeast 
-        title={'Ram'}
-        imgUrl={'https://berryhillranch.com/wp-content/uploads/2019/05/Hawaiian_Black_Sheep-square-300x300.jpg'}
-        description={'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Fermentum iaculis eu non diam phasellus vestibulum. Mattis aliquam faucibus purus in massa tempor nec feugiat.'}
-        />
-        <HornedBeast 
-        title={'Bull'}
-        imgUrl={'https://s3-us-west-2.amazonaws.com/theathletic-team-logos/team-logo-73-300x300.png'}
-        description={'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Erat imperdiet sed euismod nisi porta lorem mollis. Nunc pulvinar sapien et ligula ullamcorper malesuada proin libero.'}
-        />
-      </div>
-    )
+      // <div>
+      //   <h1>Beasts: {this.props.message}</h1>
+      //   <HornedBeast 
+      //   title={'Ram'}
+      //   imgUrl={'https://berryhillranch.com/wp-content/uploads/2019/05/Hawaiian_Black_Sheep-square-300x300.jpg'}
+      //   description={'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Fermentum iaculis eu non diam phasellus vestibulum. Mattis aliquam faucibus purus in massa tempor nec feugiat.'}
+      //   />
+      //   <HornedBeast 
+      //   title={'Bull'}
+      //   imgUrl={'https://s3-us-west-2.amazonaws.com/theathletic-team-logos/team-logo-73-300x300.png'}
+      //   description={'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Erat imperdiet sed euismod nisi porta lorem mollis. Nunc pulvinar sapien et ligula ullamcorper malesuada proin libero.'}
+      //   />
+      // </div>
```

Comment:
Nice work on this!

Feedback here: Remove commented out code before merge (unless of course you need this, then please ignore me. Otherwise, poof! Unused code be gone!)